scripts/variantstore/tieout/GIAB_TIEOUT.md

```diff
@@ -0,0 +1,174 @@
+${BASE_CMD} -b truth/HG003_GRCh38_1_22_v4.2.1_benchmark.vcf.gz -e truth/HG003.gvs.evaluation.bed -c BI_HG003.${SOURCE}.chr20.maxas.vcf.gz -o BI_HG003_${SOURCE}${SUFFIX}
+${BASE_CMD} -b truth/CHM.full.38.vcf.gz -e truth/CHM.gvs.evaluation.bed -c SYNDIP.${SOURCE}.chr20.maxas.vcf.gz -o syndip_${SOURCE}${SUFFIX}
+```
+
+The do the same thing but use all records
+```
+SOURCE="gvs"
+BASE_CMD="rtg vcfeval --region chr20 --all-records --roc-subset snp,indel  --vcf-score-field=INFO.MAX_AS_VQSLOD -t human_REF_SDF"
+SUFFIX="_roc_all"
+
+${BASE_CMD} -b truth/HG001_GRCh38_GIAB_highconf_CG-IllFB-IllGATKHC-Ion-10X-SOLID_CHROM1-X_v.3.3.2_highconf_PGandRTGphasetransfer.vcf.gz \
+-e truth/HG001.gvs.evaluation.bed \
+-c NA12878.${SOURCE}.chr20.maxas.vcf.gz -o NA12878_${SOURCE}${SUFFIX}
+
+${BASE_CMD} -b truth/HG002_GRCh38_1_22_v4.2.1_benchmark.vcf.gz -e truth/HG002.gvs.evaluation.bed -c BI_HG002.${SOURCE}.chr20.maxas.vcf.gz -o BI_HG002_${SOURCE}${SUFFIX}
+${BASE_CMD} -b truth/HG002_GRCh38_1_22_v4.2.1_benchmark.vcf.gz -e truth/HG002.gvs.evaluation.bed -c UW_HG002.${SOURCE}.chr20.maxas.vcf.gz -o UW_HG002_${SOURCE}${SUFFIX}
+${BASE_CMD} -b truth/HG003_GRCh38_1_22_v4.2.1_benchmark.vcf.gz -e truth/HG003.gvs.evaluation.bed -c BI_HG003.${SOURCE}.chr20.maxas.vcf.gz -o BI_HG003_${SOURCE}${SUFFIX}
+${BASE_CMD} -b truth/CHM.full.38.vcf.gz -e truth/CHM.gvs.evaluation.bed -c SYNDIP.${SOURCE}.chr20.maxas.vcf.gz -o syndip_${SOURCE}${SUFFIX}
+```
+
+## Gather Precision and Sensitivity numbers
+
+The columns here are: filename, type, FPs, FNs, precision, sensitivity
+
+```
+for type in "snp" "indel"
+do
+    for suffix in "_all" "_filtered"
+    do
+      for f in $(ls -1 *${suffix}/${type}*.tsv.gz); do
+        d=$(cat $f | gunzip | tail -1 | cut -f3,5,6,7)
+        s=$(echo $f | cut -d"/" -f1 )
+        echo -e "$s\t$type\t$d"
+      done
+    done
+done
+
+```
+## View ROCs
+
+You can use wildcards to pull in multiple datasets into the graphical viewer. For example
+
+```
+rtg roc NA12878_*_roc*/snp_roc.tsv.gz 
+rtg roc NA12878_*_roc*/indel_roc.tsv.gz 
+```
+
+## Appendix A: Processing WARP results against gvs_sci_tieout_acmg_cohort workspace
+
+Starting with the output of FinalGatherVcf run from the WDLs and inputs in the `wdl/legacy` directory.
+
+If this was run on the specops cromwell you can get that final VCF with:
+
+```
+WORKFLOW_ID=49f5c0a5-dbfb-4b05-b293-11ddadeae8e5
+gsutil -m cp gs://broad-dsp-spec-ops-cromwell-execution/JointGenotyping/${WORKFLOW_ID}/call-FinalGatherVcf/warp_tieout_acmg_cohort_v1.vcf.gz* .
+```
+
+Next is to subset to chr20, and the select out the 5 truth samples into their own VCFs.  Assuming a gathered initial VCF of "warp_tieout_acmg_cohort_v1.vcf.gz"
+
+```
+bcftools view -O z warp_tieout_acmg_cohort_v1.vcf.gz chr20 > warp_tieout_acmg_cohort_v1.chr20.vcf.gz
+tabix warp_tieout_acmg_cohort_v1.chr20.vcf.gz
+
+INPUT_VCF=warp_tieout_acmg_cohort_v1.chr20.vcf.gz
+SOURCE=warp
```

Comment:
does this need to be `SOURCE="warp"` ? and does INPUT_VCF assignment also need quotes? or is this one of those bash things that doesn't matter if there's no spaces?

Reply:
right -- only needed for quoting special characters, but I changed it just to be consistent